rosco/rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/api/BakeRequest.groovy

```diff
@@ -40,7 +39,7 @@ class BakeRequest {
   @Schema(description= "A generated UUID which will be used to identify the effective packer bake", accessMode = Schema.AccessMode.READ_ONLY)
   String request_id = UUID.randomUUID().toString()
   String user
-  @Schema(description="The package(s) to install, as a space-delimited string") @JsonProperty("package") @SerializedName("package")
+  @Schema(description="The package(s) to install, as a space-delimited string") @JsonProperty("package")
```

Comment:
How is this related to retrofit1?

Reply:
I added gson dependency so that any plugins still using gson for conversions would not break.